src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -289,13 +296,18 @@ const RequestDetails = ({
   ];
 
   const activeTabs = tabs.filter(t => t.visible);
+  const closePanel = () => dispatch(hideRequestDetails());
 
   useEffect(() => {
     if (!activeTabs.find(t => t.id === activeTab)) {
       setActiveTab(DEFAULT_TAB);
     }
   }, [activeTab, activeTabs]);
 
+  if (!(loadedRegions && request && getPointIsInLoadedRegion(loadedRegions, request.point.point))) {
```

Comment:
I don't have a concrete suggestion here, but I think we should keep an eye on how many places stuff like this (check if we have changed our trim region) creeps in. I suspect there might be a more centralized way of doing these things to avoid future shotgun surgery.